### PR TITLE
Mahout 1605

### PR DESCRIPTION
**Pull request by avati**
Rebased #41 and resolved conflicts. Tests pass on laptop and good for merge.

### CODE REVIEW

File: mr/src/test/java/org/apache/mahout/classifier/df/tools/VisualizerTest.java [hunk 1]
```diff
@@ -17,9 +17,7 @@
 
 package org.apache.mahout.classifier.df.tools;
 
-import java.util.List;
-import java.util.Random;
-
+import com.google.common.collect.Lists;
```
Comment (smarthi) [inline mr/src/test/java/org/apache/mahout/classifier/df/tools/VisualizerTest.java]: Desist from using Guava collections now that we r on Java 7.  This import can be dispensed with.

Comment (avati) [inline mr/src/test/java/org/apache/mahout/classifier/df/tools/VisualizerTest.java]: Updated PR, please check https://github.com/avati/mahout/commit/2bc4098b244ef182b29cba53e6eabb1f6524a29d

Comment (avati) [inline mr/src/test/java/org/apache/mahout/classifier/df/tools/VisualizerTest.java]: Not sure why this PR still says 2 commits, there should be 3 now. Can you take the third commit manually?

Comment (smarthi) [inline mr/src/test/java/org/apache/mahout/classifier/df/tools/VisualizerTest.java]: u should commit with the same message as previous one, that way the same PR
is updated

Comment (avati) [inline mr/src/test/java/org/apache/mahout/classifier/df/tools/VisualizerTest.java]: I am somehow not able to make the changes get reflected in this PR. Let me close this and open a new PR fresh.

**Pull request closed without merging**